scripts/build-catalogue.in

```diff
@@ -108,8 +108,12 @@ with TemporaryDirectory() as tmp:
         fd.write(cmake)
     shutil.copy2(f'@ARB_INSTALL_DATADIR@/BuildModules.cmake', tmp)
     shutil.copy2(f'@ARB_INSTALL_DATADIR@/generate_catalogue', tmp)
-    sp.run('cmake ..', shell=True, check=True, capture_output=not verbose)
-    sp.run('make',     shell=True, check=True, capture_output=not verbose)
+    if verbose:
+        sp.run('cmake ..', shell=True, check=True, stdout=sp.PIPE, stderr=sp.PIPE)
+        sp.run('make',     shell=True, check=True, stdout=sp.PIPE, stderr=sp.PIPE)
```

Comment:
For bonus credits: add a `try` block around the whole affair, as `check=True` will throw.
But that can be cleaned-up later as well, the whole script could benefit from an overhaul.

Reply:
That's however a change from the original script, which was not the point of this fix (which was to remove a flag because it was not in Python 3.6).

Shall we have actual changes in another PR?

Reply:
That's completely fine, as it was just something I noticed in passing.

Reply:
I have proposed a fix for this in #1679. Ignore it here and I'll merge when this PR is merged.